client/gutenberg/extensions/business-hours/index.js

```diff
@@ -0,0 +1,83 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+import './editor.scss';
+import HoursList from './components/HoursList';
+
+/**
+ * Block Registrations:
+ */
+
+export const name = 'business-hours';
+
+export const settings = {
+	title: __( 'Business Hours', 'random-blocks' ),
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
Lets use the `import renderMaterialIcon from 'gutenberg/extensions/presets/jetpack/utils/render-material-icon'; `here